ksonnet-gen/ksonnet/emit.go

```diff
@@ -68,6 +70,16 @@ func (root *root) emit(m *indentWriter) {
 		group.emit(m)
 	}
 
+	m.writeLine("local hidden = {")
```

Comment:
NB -- it seems cleaner to me to have the hidden items be inside the api group and version instead of at the top level with the need to recreate API group and version.  That would facilitate breaking things into files in the future.

But to do that you couldn't use `local` and would have to put them under a ~visible property.  You could name that `__hidden` but and have autocomplete ignore any double underscore names.

But -- whatever -- this works for now and we can visit this later if necessary.